### DISCUSSION TITLE
Error when using PlotStationary

### DISCUSSION THREAD

**Discussion by HVinther** [bmcclintock/momentuHMM#144] [Q&A]
I get the following error when running PlotStationary on my model

error in qr.default(t(const)) : 
  NA/NaN/Inf in foreign function call (arg 1)
 
My model can be seen below. I've tried modelling the effects of both sea surface temperature and distance to -1 degree celsius isoline with splines. I get the error both, when trying this on the miHMM-, the HMMfits and the miSum object.

` 
ModelSSTDTI<-MIfitHMM(miData = MIdata, ncores = 6, alpha = 0.95, nSims = nSims, poolEstimates = TRUE, fit = TRUE, nbStates = 2, formula = ~ splines::ns(SST, df = 5)+splines::ns(DTI, df = 5), dist = list(step = "gamma", angle = "vm"), Par0 = list(step = c(800,2500,1000,1000),angle = c(2,10)), estAngleMean = list(angle = FALSE))`

When I model the effect of just one covariate I get no such error. 

I've tried running qr on the hessians of the HMMfits object, where no problem seems to occour.

The same error also shows up on the state trasition probability plot (plot number 3), when plotting the miHMM object. The functions stationary and getTrProbs seems to work without a problem.

(I've tried including zipped .Rdata files of the model and data, but get an "is not included in the list" argument, when trying to.)

**Comment by bmcclintock** [comment]
I'd need a minimal reproducible example to diagnose exactly what is going on, but I'd guess that you have state transition probability parameters that aren't identifiable or are being estimated near a boundary. Have you examined the model output? If the output seems OK, then send me a minimal reproducible example so I can try and diagnose.
